src/api/movie_api.ts

```diff
@@ -0,0 +1,145 @@
+import {Filters} from 'src/utils/filters'
```

Comment:
It's best to be consistent with file names. In the case of functions or constants, camel case (e.g. `getMoreDataFromEndpoint.ts`)

Reply:
Changed it to `filters`